12 nav bar component

ArthiRagothaman opened the pull request:
Updated the NavBar component with icons, styles.
Included the hero-icons and added usestate hooks.

ReallyBoard123 commented on src/layouts/NavigationsBar.tsx:
```diff
@@ -0,0 +1,92 @@
+          onClick={() => setSelected(2)}
+        />
+
+        {/* <NavLink
```
kindly delete all the commented block and chunks of code from the file

ReallyBoard123 commented on src/components/NavigationBar.tsx:
```diff
@@ -3,13 +3,12 @@ import { NavLink } from "react-router-dom";
 export function NavigationBar() {
```
where will this be used in the app?

ReallyBoard123 replied:
Has this been resolved? This seems to be unaddressed. Where is this going to be used?

ReallyBoard123 commented on src/layouts/NavigationsBar.tsx:
```diff
@@ -0,0 +1,92 @@
+export default function Navigationsbar() {
+  const [selected, setSelected] = useState(0);
+
+  return (
```
please see if hardcoding w and h can be avoided! please use pre-set tailwind config for these. 

or use w/h using tailwind parameters

ReallyBoard123 commented on src/layouts/NavigationsBar.tsx:
```diff
@@ -0,0 +1,92 @@
+
+  return (
+    <div className=" w-[375px] h-[88px] bg-dark ">
+      <nav className="flex items-center justify-center mt-5 mx-3 ">
```
Is this possible to map over a set of icons and write the clsx tailwind classes once?

ReallyBoard123 commented on src/layouts/NavigationsBar.tsx:
```diff
@@ -0,0 +1,92 @@
+import clsx from "clsx";
+import { useState } from "react";
+
+export default function Navigationsbar() {
```
please name the functions using the conventions

ReallyBoard123 reviewed (changes requested):
please resolve the comments :) You are almost there!!

ReallyBoard123 reviewed (changes requested):
- all the previous comments have not been resolved/addressed
- please also link the issue that this pull request is addressing
- please go through the issue that it references, and checkbox the tasks that have been completed in these files

ArthiRagothaman commented:
In response to ur comments:
Deleted the commented lines and updated the naming conventions.
Attached the branch name in development section.
All changes done.

ReallyBoard123 commented on src/layouts/Navigationsicons.tsx:
```diff
@@ -0,0 +1,64 @@
+import clsx from "clsx";
+import { useState } from "react";
+
+export default function Navigationsicons() {
```
please follow the correct naming conventions for functions

ReallyBoard123 commented on src/layouts/Navigationsicons.tsx:
```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+
+export default function Navigationsicons() {
+  const [selected, setSelected] = useState(0);
```
there shouldn't be any need for a useState here, since you can use className={({ isActive }) in Navlink. On further examination, a duplicate of the file actually exists in the folder structure by the name RootLayout, which already has a solution commented out in it. Please see if you can use that file, rename it properly and make the icons work. The icons are not supposed to just change the styling when clicked, but supposed to "navigate" from page A to page B. and when the navigation happens, styling changes

ReallyBoard123 reviewed (changes requested):
Please try to look into how the navigation layout will function. It might be easier to understand the coding part when you can visualize its function or usage. There are some empty files in the branch; if they are empty and have no purpose for a future usage, please delete them.

ArthiRagothaman commented:
Task#2 
Required to create separate NavigationBar component. I found it was easier to have directly write the code in NavigationLayout itself.

ReallyBoard123 reviewed (approved):
Hey Arthi!! Job well done!! I am so happy to see that you got through all the tasks, and made great progress in such little time. I look forward to seeing such complete pull requests in future as well!
